src/axom/slam/Map.hpp

```diff
@@ -515,8 +522,8 @@ bool Map<T, S, IndPol, StrPol>::isValid(bool verboseOutput) const
   return bValid;
 }
 
-template <typename T, typename S, typename IndPol, typename StrPol>
-void Map<T, S, IndPol, StrPol>::print() const
+template <typename T, typename S, typename IndPol, typename StrPol, typename IfacePol>
+void Map<T, S, IndPol, StrPol, IfacePol>::print() const
```

Comment:
I can't say I am a fan of this level of shortening the name. It makes it hard to read. I would much prefer `InterfacePolicy`. I realize this is just following what is there but it also doesn't match the rest of the coding style.

Reply:
Thanks @white238. I agree that the brevity here doesn't match other parts of slam and would be in favor of more overall consistency throughout slam. It's ok with me to save it for a future PR though.

Reply:
Agreed in full -- I'll change it to be consistent with the rest of Slam in a future PR.